### PR TITLE
Db 827 separation of integration and unit tests

### PR DESCRIPTION
**Pull request by nmonga91**

### CODE REVIEW

File: tests/integration/baseTestValidator.py [hunk 1]
```diff
@@ -214,8 +214,7 @@ def uploadFile(cls, filename, user):
         bucketName = CONFIG_BROKER['aws_bucket']
         regionName = CONFIG_BROKER['aws_region']
 
-        path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
-        fullPath = path + "/" + filename
+        fullPath = os.path.join(CONFIG_BROKER['path'], "tests", "integration", "data", filename)
```
Comment (cmc333333) [inline tests/integration/baseTestValidator.py]: Wow, yes this is much much cleaner.

File: tests/integration/fileTypeTests.py [hunk 2]
```diff
@@ -101,11 +101,14 @@ def load_definitions(interfaces, force_tas_load):
         SchemaLoader.loadAllFromPath(join(CONFIG_BROKER["path"],"dataactvalidator","config"))
         SQLLoader.loadSql("sqlRules.csv")
         # Load domain values tables
-        loadDomainValues(join(CONFIG_BROKER["path"],"dataactvalidator","config"),join(CONFIG_BROKER["path"],"tests","sf_133.csv"),join(CONFIG_BROKER["path"],"tests","program_activity.csv"))
+        loadDomainValues(
+            join(CONFIG_BROKER["path"], "dataactvalidator", "config"),
+            join(CONFIG_BROKER["path"], os.path.join("tests", "integration", "data"), "sf_133.csv"),
+            join(CONFIG_BROKER["path"], os.path.join("tests", "integration", "data"), "program_activity.csv"))
```
Comment (cmc333333) [inline tests/integration/fileTypeTests.py]: So you need the separate `os.path.join`s here? I'd think the initial `join` would do the trick.

File: tests/unit/dataactcore/test_status_codes.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        assert StatusCode.CLIENT_ERROR == 400
+
+    def test_status_login_required(self):
+        assert StatusCode.LOGIN_REQUIRED == 401
+
+    def test_status_internal_error(self):
+        assert StatusCode.INTERNAL_ERROR == 500
+
+    def test_intentional_failure(self):
+        assert StatusCode.OK == 201
```
Comment (cmc333333) [inline tests/unit/dataactcore/test_status_codes.py]: NBD, but we don't really need a class here; we certainly don't need the setup/teardown methods.

Comment (nmonga91) [inline tests/unit/dataactcore/test_status_codes.py]: This is now resolved

File: tests/integration/fileTypeTests.py (hunk 2 above)
Comment (nmonga91) [inline tests/integration/fileTypeTests.py]: I didn't change this portion, Becky had done this but it looks like first join handles the basePath parameter in loadDomainValues while the others lead to a specific test file that's in a completely different directory

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
[run tests]

**Comment by cmc333333** [comment]
Excited to see it!

**Pull request merged**